massa-executed-ops/src/executed_ops.rs

```diff
@@ -39,6 +39,8 @@ pub struct ExecutedOps {
     pub ops: PreHashSet<OperationId>,
```

Comment:
how about just changing this into a `PreHashMap<OperationId, Result<(), String>>` instead of having another container that we must not forget to update/clear ?